src/debugging/DockerDebugConfigurationProvider.ts

```diff
@@ -170,4 +170,51 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
             }
         }
     }
+
+    /**
+     * Copies the first debug configuration from the array to the second one.
+     */
+    private copyDebugConfiguration(fromArray: DockerDebugConfiguration[], to: DockerDebugConfiguration): void {

```

Comment:
Does it not work to just do something like `to = fromArray[0]`?

Reply:
(Or, the equivalent in the above function)

Reply:
good suggestion, it does work actually